### PR TITLE
Scaling the AWS cluster paragraph added

### PR DESCRIPTION
**Pull request by tolleiv**
Hi,
I found answers to that question on StackOverflow [1](http://stackoverflow.com/questions/31442580/add-nodes-to-kubernetes-cluster-on-aws),[2](http://stackoverflow.com/questions/36665326/how-to-add-a-kubernetes-node-to-the-existing-cluster-on-aws?rq=1) and assume that its saving others some time if it's part of the documentation.
Cheers

### CODE REVIEW

**Comment by tolleiv** [comment]
I signed it!

File: docs/getting-started-guides/aws.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ The "Guestbook" application is another popular example to get started with Kuber
 
 For more complete applications, please look in the [examples directory](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/)
 
+## Scaling the cluster
+
+Adding and removing nodes through `kubectl` is not supported. You can still scale the amount of nodes through manual adjustments of the Auto Scaling Group which was created during the installation. In cause you increase the number of nodes, the new node will automatically join the cluster once it's ready.
```
Comment (johndmulhausen) [inline docs/getting-started-guides/aws.md]: Where are manual adjustments made? Add that info here. 

Also, you can remove the last sentence; that's widely expected behavior.

**Comment by tolleiv** [comment]
Adjusted the text and added link to the Amazon documentation.

**Comment by pwittrock** [comment]
@justinsb Mind taking a look?

**Comment by kelseyhightower** [comment]
This looks good to me. Touching the autoscaling group is the only way to really do this.

**Pull request merged**